src/4_run_evaluation.py

```diff
@@ -706,6 +704,46 @@ def enumerate_tasks(self, max_instance_count: int):
         print("----------------------------------------------")
         print(f'Field statistics per task: {task_field_statistics}')
 
+    def enumerate_comprehensive_tests(self, max_instance_count: int):
```

Comment:
I don't see where you call this function.

Reply:
yeah this function isn't being called right now, still a wip. that's a stub, small progress. this PR was mostly for the clear_text fix for my Mac devices in case a mac user wanted to run tests locally and continually fails the HTER task